src/MDMPerforming.h

```diff
@@ -148,6 +148,30 @@ NS_SWIFT_NAME(ComposablePerforming)
 
 @end
 
+/** Specifics for a named plan performer to allow named plans to be added and removed. */
+NS_SWIFT_NAME(NamedPlanPerforming)
+@protocol MDMNamedPlanPerforming <MDMPerforming>
+
+/**
+ Provides the performer with a plan and an associated name.
+
+ @param plan The plan that required this type of performer.
+ @param name The name by which the plan can be identified.
+ */
+- (void)addPlan:(nonnull id<MDMPlan>)plan
+          named:(nonnull NSString *)name
+  NS_SWIFT_NAME(addPlan(plan:named:));
```

Comment:
Hm, this API brings up a good point. I agree here that `addPlan` is more consistent with `removePlan`, so in order to avoid the redundancy of `addPlan(plan:named:)` we can make the first argument non-named: `addPlan(_:named:)`.

This will allow us to invoke the method like so: `addPlan(SomePlan, named:"name")` rather than `addPlan(plan:SomePlan, named:"name")`.